std/random.d

```diff
@@ -1542,8 +1559,16 @@ if ((isIntegral!(CommonType!(T1, T2)) || isSomeChar!(CommonType!(T1, T2))) &&
 
 /**
 Generates a uniformly-distributed number in the range $(D [T.min,
-T.max]) for any integral type $(D T). If no random number generator is
-passed, uses the default $(D rndGen).
+T.max]) for any integral or character type $(D T). If no random
+number generator is passed, uses the default $(D rndGen).
+
+Params:
+    urng = (optional) random number generator to use;
+           if not specified, defaults to $(D rndGen)
+
+Returns:
+    Random variate uniformly distributed across all the different
```

Comment:
On 06/01/15 05:22, Daniel Murphy wrote:
> I'd object to this too, but the summary makes it clear enough.

What's your objection?  I'll happily tweak things if it seems a good idea (I 
need to revisit the module anyway in order to document `Params:` for struct 
methods).